package.json

```diff
@@ -15,19 +15,24 @@
       "url": "https://github.com/jesusfchavarro"
     }
   ],
+  "main": "src/index.tsx",
```

Comment:
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#main
el main debe ser el punto de entrada al paquete y por ende debe ser un archivo js, para que todo el mundo lo pueda utilizar el paquete debe ser entregado compilado. Si se quieren agregar soporte a ts se necesita crear[ `definition files` ](https://stackoverflow.com/questions/21247278/about-d-ts-in-typescript) con el compilador y asignarlo al campo 'types' https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html